gammapy/irf/energy_dispersion.py

```diff
@@ -626,6 +626,48 @@ def offset(self):
         return self.data.axis('offset')
 
     @classmethod
+    def _from_gaus(cls, e_true, migra, bias, sigma, offset=None):
+        """Create Gaussian `EnergyDispersion2D` matrix.
+
+         The output matrix will be Gaussian in (e_true / e_reco)
+         The output matrix is flat in offset
+         TODO: add option to have log normal distribution
+
+         Parameters
+         ----------
+         e_true : `~astropy.units.Quantity`, `~gammapy.utils.nddata.BinnedDataAxis`
+             Bin edges of true energy axis
+         migra : `~astropy.units.Quantity`, `~gammapy.utils.nddata.BinnedDataAxis`
+             Bin edges of migra axis
+         sigma : float, optional
+             RMS width of Gaussian energy dispersion, resolution
+         offset : `~astropy.units.Quantity`, `~gammapy.utils.nddata.BinnedDataAxis`, optional
+             Bin edges of offset
+         """
+        if offset is None:
+            offset = np.linspace(0.,3.,10)*u.deg
```

Comment:
I know we have defaults in a lot of places that really only work well for HESS, like 3 deg FOV size here.
Probably we should remove those, no?
I.e. the caller must be explicit, but then we have no HESS-specific hardcoded stuff.
You know, be CTA friendly.